src/schema/v2/artwork/index.ts

```diff
@@ -893,8 +893,10 @@ export const ArtworkType = new GraphQLObjectType<any, ResolverContext>({
         description: "Is this work only available for shipping domestically?",
         resolve: (artwork) => {
           return Boolean(
+            // should we factor in artwork.artsy_shipping_domestic ?? probably yeah?
             artwork.domestic_shipping_fee_cents &&
```

Comment:
I believe this needs to be (artwork.domestic_shipping_fee_cents != null || artwork.artsy_shipping_domestic) && !(international_shipping_fee_cents !== null || artsy_shipping_international).

Or we can be even more explicit:
```
const domesticShippingDefined? = artwork.domestic_shipping_fee_cents != null || artwork.artsy_shipping_domestic
const internationalShippingDefined? = artwork.international_shipping_fee_cents != null || artwork.artsy_shipping_international
return domesticShippingDefined? && !internationalShippingDefined?
```


The point is we need to account for artsy shipping in both domestic and international and all the mix/match cases I believe.

One question is if no shipping defined at all - everything is nil/false - this method will return `false`. That is current and intended behaviour I think~

Reply:
> One question is if no shipping defined at all - everything is nil/false - this method will return false. That is current and intended behaviour I think~

Sounds like this is correct and inline with the previous behavior.

Addressed and added specs!!!